package.json

```diff
@@ -46,7 +46,7 @@
     "jest-coverage": "yarn run jest -- --coverage",
     "lint": "eslint . --cache && yarn run lint-docs",
     "lint-docs": "eslint --config ./.eslintrc-docs.json --no-ignore --ext md ./docs/*.md",
-    "prettier": "prettier --write 'packages/*/src/**.js' --single-quote --trailing-comma=all --bracket-spacing=false",
+    "prettier": "prettier --write 'packages/*/src/**/*.js' --single-quote --trailing-comma=all --bracket-spacing=false",
```

Comment:
Yes, eventually. But for now we need to get the repo to be consistent with prettier's output and I'd like to limit that to the `packages/` folder first. I might spend some time on that next week.